terraform/modules/prometheus/data.tf

```diff
@@ -1,2 +1,6 @@
 data "aws_region" "current" {}
 data "aws_availability_zones" "current" {}
+
+data "aws_ecr_repository" "prometheus" {
```

Comment:
Usage should reference the output from the management remote state, not a `data`

Reply:
Resolved